Ft testing 187303859

RWEMAREMY opened the pull request:
### issue in referencing?

### Questions:

#### What does this PR do?

testing with mocha locally

#### Description of Task to be completed?

- Using mocha with chai in testing


#### How should this be manually tested?

- use **npm run test** to test

#### Any background context you want to provide?

-dummy data are provided before having the real data ,use real data while testing and lets keep it on track of 85% and above
#### What are the relevant pivotal tracker/Trello stories?

- id of the project
#187303859

#### Questions:

RWEMAREMY committed:
testing 93%

RWEMAREMY committed:
conflict removed

RWEMAREMY committed:
chore(testing):changing name of my branch
-changing name from be-testing-187303859 to ch-testing-187303859
[fixed #187303859]

RWEMAREMY committed:
ft(testing):changes in config.js

-changed postgres to  dialect: process.env.DIALECT

[delivered #187303859]

TuyishimireEric reviewed (approved):
# Status: APPROVED 🟢  
### Hi  @RWEMAREMY  👋 👋 

#### I'm @TuyishimireEric,
Good job on setting  Test Locally🥇 

![done](https://media.giphy.com/media/l1uga8ObPBnIBoXUQ/giphy.gif)

Your task is complete!  Congratulations! 🎉 

### Cheers and Happy coding!👏👏👏

Ruthuwamahoro commented:
Hello, @RWEMAREMY.
Ruth here! You have done a fantastic work. Keep up.

fab-ryan commented on .gitignore:
```diff
@@ -1,3 +1,5 @@
 .env
 node_modules
+coverage
+.nyc_output
```
new line here

Ruthuwamahoro reviewed (approved):
Hello @RWEMAREMY .Ruth Here.Congraturations.You did an amazing job

fab-ryan commented on src/app.ts:
```diff
@@ -1,16 +1,16 @@
 import router from "./routers";
-import * as swaggerDocument from '../swagger.json';
-import swaggerUi from 'swagger-ui-express';
+
+
```
why swagger documentation have been removed

fab-ryan commented on src/app.ts:
```diff
@@ -1,16 +1,16 @@
 app.use(cors());
 app.use(router);
 
-app.use('/swagger',swaggerUi.serve,swaggerUi.setup(swaggerDocument))
 
```
Don't remove someone else's codes

RWEMAREMY replied:
my bad, let me fix it

fab-ryan commented on src/routers/dummyRoute.ts:
```diff
@@ -1,5 +1,5 @@
 import { Router } from 'express';
-import { dummyRequest } from '../controllers/dummyController';
+ import dummyRequest from '../controllers/dummyController';
 
```
Please don't change someone else's structure code here you have imported it as default while it's was not default imports

fab-ryan commented on tsconfig.json:
```diff
@@ -106,5 +106,6 @@
-}
+  "include": ["src/*/.ts", "src/config/config.js", "test/*/.ts", "swagger.json"],
+  "exclude": ["node_modules"]
+}
```
new line please

fab-ryan reviewed (changes requested):
Please don't change someone else's codes work on those comments

RWEMAREMY committed:
ft(testing):changes in coments of reviewers

-changed postgres to  dialect: process.env.DIALECT

[delivered #187303859]

fab-ryan commented on src/routers/dummyRoute.ts:
```diff
@@ -1,5 +1,5 @@
 import { Router } from 'express';
-import { dummyRequest } from '../controllers/dummyController';
+import { dummyRequest } from "../controllers/dummyController"
```
please use EsLint you not prittier formatter

fab-ryan reviewed (changes requested):
Change that miner issuer then I will merge it

fab-ryan reviewed (changes requested):
Also I did tell you to remove nyc report and coverage, but I still see it there why??

RWEMAREMY commented on .gitignore:
Here they have removed, take a look at my .gitignore

RWEMAREMY commented on .gitignore:
Take a look at my gitignore, I have removed them lately.

fab-ryan replied:
to add them in gitignore doesn't mean you have removed them on remote do research how to remove file/folder from git

fab-ryan replied:
OOHH I saw the reason please you have two branches which is identical remove that one with moch

RWEMAREMY replied:
Let do it as soon as possible

RWEMAREMY committed:
ft(testing):changes basing on coments of reviewer's insight

-changed in routers/dummyRouter.ts from prittier to Eslint Format

[delivered #187303859]

fab-ryan reviewed (approved):
Good work
